testsuite/manualmode/src/test/java/org/wildfly/core/test/standalone/mgmt/RemoveManagementInterfaceTestCase.java

```diff
@@ -184,4 +192,59 @@ static ModelControllerClient getHttpModelControllerClient() {
         }
         return client;
     }
+
+    private static void reloadServer(final ModelControllerClient client) {
+        executeReload(client);
+        waitForLiveServerToReload(TimeoutUtil.adjust(30000)); //30 seconds
+    }
+
+    private static void executeReload(final ModelControllerClient client) {
+        ModelNode operation = new ModelNode();
+        operation.get(OP_ADDR).setEmptyList();
+        operation.get(OP).set("reload");
+        try {
+            ModelNode result = client.execute(operation);
+            Assert.assertEquals("success", result.get(ClientConstants.OUTCOME).asString());
+        } catch (IOException e) {
+            final Throwable cause = e.getCause();
+            if (!(cause instanceof ExecutionException) && !(cause instanceof CancellationException) && !(cause instanceof SocketException)) {
+                throw new RuntimeException(e);
+            } // else ignore, this might happen if the channel gets closed before we got the response
+        } finally {
+            safeCloseClient(client);//close existing client
+        }
+    }
+
+    private static void safeCloseClient(ModelControllerClient client) {
+        try {
+            if (client != null) {
+                client.close();
+            }
+        } catch (final Exception e) {
+            LOGGER.warnf(e, "Caught exception closing ModelControllerClient");
+        }
+    }
+
+    private static void waitForLiveServerToReload(int timeout) {
+        long start = System.currentTimeMillis();
+        ModelNode operation = new ModelNode();
+        operation.get(OP_ADDR).setEmptyList();
+        operation.get(OP).set(READ_ATTRIBUTE_OPERATION);
+        operation.get(NAME).set("server-state");
+        while (System.currentTimeMillis() - start < timeout) {
+            try (ModelControllerClient liveClient = getNativeModelControllerClient()){
+                ModelNode result = liveClient.execute(operation);
+                if ("running".equals(result.get(RESULT).asString())) {
+                    return;
+                }
+            } catch (IOException e) {
+            }
+            try {
+                Thread.sleep(100);
+            } catch (InterruptedException e) {
+            }
+
+        }
+        fail("Live Server did not reload in the imparted time.");
+    }
```

Comment:
This util stuff looks like what's already in the ServerReload class in testsuite/shared.